core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -115,6 +117,17 @@ public boolean isValid() {
         return getProjectConfig() != null;
     }
 
+    @Override
+    public void close() throws IOException {
```

Comment:
Would love guidance on what sort of tests make sense here.

Reply:
* Test that close gets called for these implementations. 
* Test that exceptions thrown during close are handled appropriately (probably just a warn)
* There's also the case that they might block indefinitely trying to close, but maybe we need to leave that up to the implementations themselves to prevent. So a comment would suffice.

Reply:
Thanks. Will respond accordingly.